### PR TITLE
Test: unit test for `FormatTimeMillis`

### PR DESCRIPTION
**Pull request by tanryberdi**
Added unit-tests for FormatTimeMillis func

ref: https://github.com/cectc/dbpack/issues/70

### Ⅰ. Describe what this PR did
Adding unit-tests for FormatTimeMillis func

### Ⅱ. Does this pull request fix one issue?
I am not sure, but i think yes

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
tests added

### Ⅴ. Special notes for reviews
this PR is ok for you, i will continue for others, let me know if you have any comments for PR

### CODE REVIEW

**Comment by dk-lockdown** [comment]
Thank you for your pr, hope you can learn more through this project. You are more than welcome to receive other tasks.

**Comment by tanryberdi** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by dk-lockdown — APPROVED** [review]
LGTM

File: pkg/misc/time_test.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package misc
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
```
Comment (dk-lockdown) [inline pkg/misc/time_test.go:4]: pls format this import block, group stdlib imports, group the third party imports.

Comment (tanryberdi) [reply pkg/misc/time_test.go:4]: man, now, i can not configure my `.gitconfig` file for multilpe account for gitlab and github

Comment (dk-lockdown) [reply pkg/misc/time_test.go:4]: you can config `~/.ssh/config`:
```
Host github.com
    HostName github.com
    IdentityFile ~/.ssh/id_rsa_github
```

Comment (tanryberdi) [reply pkg/misc/time_test.go:4]: i have those on my `~/.ssh/config`
```
# GitLab.com
Host gitlab.com
  PreferredAuthentications publickey
  IdentityFile ~/.ssh/id_ed25519

# Personal github account
Host github.com
   HostName github.com
   IdentityFile ~/.ssh/id_rsa_github
```

Comment (tanryberdi) [reply pkg/misc/time_test.go:4]: > you can config `~/.ssh/config`:
> 
> ```
> Host github.com
>     HostName github.com
>     IdentityFile ~/.ssh/id_rsa_github
> ```

for mine like this, but i got error from github actions, sorry for my bad, this one is first time for me

Comment (tanryberdi) [reply pkg/misc/time_test.go:4]: man, i fixed my bad, thanks

**Comment by tanryberdi** [comment]
i wrote unit tests for `func FormatDate(tsMillis uint64) string`

**Comment by dk-lockdown** [comment]
pls check the ci results, and use `time.FixedZone` fix time zone issue.

**Comment by tanryberdi** [comment]
> pls check the ci results, and use `time.FixedZone` fix time zone issue.

I used `UTC()` for fix time zone issue, wdyt? will it work for you?

**Comment by dk-lockdown** [comment]
it's ok

**Comment by dk-lockdown** [comment]
`CurrentTimeMillis()` and `CurrentTimeNano()` should set time zone too

**Comment by hzliangbin** [comment]
LGTM

**Comment by tanryberdi** [comment]
btw, when i `make test` in repo, i got some test fails in `github.com/cectc/dbpack/pkg/optimize `, how can i fix it for my local repo?

**Comment by dk-lockdown** [comment]
```
git remote add upstream  git@github.com:CECTC/dbpack.git
git fetch upstream
git pull --rebase upstream dev
git push -f
```

**Comment by tanryberdi** [comment]
> 

did not fix for me, 
```
--- FAIL: TestOptimizeInsert (0.00s)
panic: permission denied [recovered]
        panic: permission denied

goroutine 22 [running]:
testing.tRunner.func1.2({0x101e69e60, 0x102c5b068})
        /usr/local/go/src/testing/testing.go:1389 +0x1c8
testing.tRunner.func1()
        /usr/local/go/src/testing/testing.go:1392 +0x384
panic({0x101e69e60, 0x102c5b068})
        /usr/local/go/src/runtime/panic.go:838 +0x204
github.com/agiledragon/gomonkey/v2.modifyBinary(0x101a21ae0, {0x140000401c8, 0x18, 0xc7e5043d10886077?})
        /Users/tanryberdi/go/pkg/mod/github.com/agiledragon/gomonkey/v2@v2.7.0/modify_binary_darwin.go:9 +0xa4
github.com/agiledragon/gomonkey/v2.replace(0x101a21ae0, 0x1400052da90?)
        /Users/tanryberdi/go/pkg/mod/github.com/agiledragon/gomonkey/v2@v2.7.0/patch.go:255 +0x8c
github.com/agiledragon/gomonkey/v2.(*Patches).ApplyCore(0x1400052da78, {0x140004cb080?, 0x1400059e148?, 0x101fb7880?}, {0x140004cb080?, 0x14000371980?, 0x14000136000?})
        /Users/tanryberdi/go/pkg/mod/github.com/agiledragon/gomonkey/v2@v2.7.0/patch.go:217 +0xe8
github.com/agiledragon/gomonkey/v2.(*Patches).ApplyMethodFunc(0x14000085a88?, {0x101e6bf60, 0x0}, {0x101ad6ddc, 0xc}, {0x101e1dca0, 0x101fb7880})
        /Users/tanryberdi/go/pkg/mod/github.com/agiledragon/gomonkey/v2@v2.7.0/patch.go:100 +0x11c
github.com/agiledragon/gomonkey/v2.ApplyMethodFunc(...)
        /Users/tanryberdi/go/pkg/mod/github.com/agiledragon/gomonkey/v2@v2.7.0/patch.go:32
github.com/cectc/dbpack/pkg/optimize.TestOptimizeInsert(0x14000503ba0)
        /Users/tanryberdi/work/open-source/dbpack/pkg/optimize/optimizer_test.go:95 +0x260
testing.tRunner(0x14000503ba0, 0x101fb7888)
        /usr/local/go/src/testing/testing.go:1439 +0x110
created by testing.(*T).Run
        /usr/local/go/src/testing/testing.go:1486 +0x300
```

**Comment by dk-lockdown** [comment]
```
cd `go env GOPATH`
git clone https://github.com/eisenxp/macos-golink-wrapper.git
mv `go env GOTOOLDIR`/link `go env GOTOOLDIR`/original_link
cp `go env GOPATH`/macos-golink-wrapper/link  `go env GOTOOLDIR`/link
chmod +x `go env GOTOOLDIR`/link
```

**Comment by tanryberdi** [comment]
> ```
> cd `go env GOPATH`
> git clone https://github.com/eisenxp/macos-golink-wrapper.git
> mv `go env GOTOOLDIR`/link `go env GOTOOLDIR`/original_link
> cp `go env GOPATH`/macos-golink-wrapper/link  `go env GOTOOLDIR`/link
> chmod +x `go env GOTOOLDIR`/link
> ```

did not work one more time, i got same fails ((

**Comment by dk-lockdown** [comment]
you can ignore it, that is not your problem, that problem may occur on mac

**Comment by tanryberdi** [comment]
> you can ignore it, that is not your problem, that problem may occur on mac

got it

**Comment by zackzhangkai** [comment]
LGTM

**Comment by dk-lockdown** [comment]
> > you can ignore it, that is not your problem, that problem may occur on mac
> 
> got it
 add `-gcflags=all=-l ` can solve this problem, like this:
```
	@go test -gcflags=all=-l -short ${PKG_LIST}
```

**Pull request merged**